roles/StackStorm.mongodb/vars/redhat_6.yml

```diff
@@ -0,0 +1,7 @@
+mongo_dependencies:
+      - python-urllib3
+      - pyOpenSSL
+      - python-pyasn1
+      - python-ndg_httpsclient
+# Extract mongodb "major.minor" version
+mongodb_major_minor_version: "{{ (mongodb_version|string)[:3] }}"
```

Comment:
Because `mongodb_major_minor_version` is now the same across the OSes, we can move it into the shared `vars/main.yml` file.

Reply:
Example based on `st2` role:

https://github.com/StackStorm/ansible-st2/blob/94e3c4f37e023b2783b0de4294f2dc154d73150f/roles/StackStorm.st2/vars/main.yml

Reply:
Addressed, and I also nolonger needed redhat_6/7 variable files either - so consolidated those back into redhat.yml

Reply:
Thanks, looks so much cleaner! :+1: